packages/experience/src/pages/ErrorPage/SupportInfo.tsx

```diff
@@ -0,0 +1,51 @@
+import { useContext } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+
+import PageContext from '@/Providers/PageContextProvider/PageContext';
+import TextLink from '@/components/TextLink';
+
+import styles from './index.module.scss';
+
+const TextLinkComponent = ({ href, text }: { readonly href: string; readonly text: string }) => (
+  <TextLink href={href}>{text}</TextLink>
+);
+
+const SupportInfo = () => {
+  const { experienceSettings } = useContext(PageContext);
+  const { t } = useTranslation();
+
+  if (!experienceSettings?.supportEmail && !experienceSettings?.supportWebsiteUrl) {
+    return null;
+  }
```

Comment:
nit: This can be moved down under line 21

Reply:
experienceSettings might be undefined.